src/UserCard.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 128
+  }
+});
+
+// component: semantic, variant: style
+export default function UserCard({ userId }) {
+  const [user, setUser] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    (async function() {
+      const res = await fetch(`https://reqres.in/api/users/${userId}`);
```

Comment:
One other nice-to-have might be a `try`/`catch` block to handle cases where the API errors out. In that case, you might want to display an error message instead of staying in the loading state. 

```js
try {
  // fetch the user, etc.
} catch (err){
  setError(err);
}
```

This assumes that you have an `error` piece of state that you only render if it's present. Let me know if that makes sense!